Integrations/integration-ArcherRSA.yml

```diff
@@ -967,46 +995,54 @@ script:
     }
 
     function getAuthToken() {
-        var result = sendRestRequest("login",
-            {
-                InstanceName: params.instanceName,
-                Username: params.credentials.identifier,
-                Password: params.credentials.password
-            },
-            false
-        );
-
-        if (result.IsSuccessful != true) {
-            throw 'Login failed. Result = ' + JSON.stringify(result);
+        var soap = createUserSessionFromInstanceSoapRequest(params.instanceName,
+                                                            params.credentials.identifier,
+                                                            params.credentials.password);
+        var SOAPAction = 'http://archer-tech.com/webservices/CreateUserSessionFromInstance';
+        var response = JSON.parse(x2j(sendSoapRequest('login', soap, SOAPAction)));
+        if (response === null || typeof response !== 'object' || response === undefined) {
+            throw 'Login failed. Result = ' + response;
```

Comment:
most probably if you reached this line, your response will not contain nothing (null). You should print the raw response of the soapRequest